acceptance/acceptance_test.go

```diff
@@ -2643,6 +2643,7 @@ include = [ "*.jar", "media/mountain.jpg", "/media/person.png", ]
 								"rebase", repoName,
 								"--run-image", runAfter,
 								"--pull-policy", "never",
+								"--force",
```

Comment:
Worth noting, the need for `--force` is a breaking change. We should call this out in the release notes.